<ui version="4.0" >
 <author>Frank Osterfeld</author>
 <class>Akregator::SettingsAppearance</class>
 <widget class="QWidget" name="Akregator::SettingsAppearance" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>General</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Article List Colors</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" >
         <widget class="QLabel" name="lbl_newArticles" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>New articles:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="KColorButton" name="kcfg_ColorUnreadArticles" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="KColorButton" name="kcfg_ColorNewArticles" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="lbl_unreadArticles" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Unread articles:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="kcfg_UseCustomColors" >
        <property name="text" >
         <string>Use custom colors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Font Size</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>8</number>
      </property>
      <property name="topMargin" >
       <number>8</number>
      </property>
      <property name="rightMargin" >
       <number>8</number>
      </property>
      <property name="bottomMargin" >
       <number>8</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="3" column="0" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="slider_mediumFontSize" >
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <number>30</number>
          </property>
          <property name="value" >
           <number>12</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval" >
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_MediumFontSize" >
          <property name="value" >
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="slider_minimumFontSize" >
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <number>30</number>
          </property>
          <property name="value" >
           <number>8</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval" >
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_MinimumFontSize" >
          <property name="value" >
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lbl_MinimumFontSize" >
        <property name="text" >
         <string>Minimum font size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lbl_MediumFontSize" >
        <property name="text" >
         <string>Medium font size:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Fonts</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>8</number>
      </property>
      <property name="topMargin" >
       <number>8</number>
      </property>
      <property name="rightMargin" >
       <number>8</number>
      </property>
      <property name="bottomMargin" >
       <number>8</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Standard font:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2" >
          <property name="text" >
           <string>Fixed font:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3" >
          <property name="text" >
           <string>Serif font:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel4" >
          <property name="text" >
           <string>Sans serif font:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="KFontComboBox" name="kcfg_StandardFont" />
        </item>
        <item>
         <widget class="KFontComboBox" name="kcfg_FixedFont" />
        </item>
        <item>
         <widget class="KFontComboBox" name="kcfg_SerifFont" />
        </item>
        <item>
         <widget class="KFontComboBox" name="kcfg_SansSerifFont" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>419</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="kcfg_UnderlineLinks" >
     <property name="text" >
      <string>&amp;Underline links</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
